PhilipChat/src/chat.jsx

```diff
@@ -0,0 +1,72 @@
+import "./chat.css";
+import React, { useState, useEffect } from "react";
+
+export default function Chat({ author }) {
+  const [messages, setMessages] = useState([]);
+  const url = "https://chat.avalon.build/chat";
+
+  async function fetchMessages() {
+    const response = await fetch(url);
+    const messagesData = await response.json();
+    setMessages(messagesData);
+  }
+
+  useEffect(() => {
+    fetchMessages();
+  }, []);
+
+  const entirePage = messages.map((m) => {
+    if (author === m.author) {
+      return (
+        <div className="message author">
+          <div className="user-author">{m.author}</div>
+          <div className="user-message">{m.message}</div>
+        </div>
+      );
+    } else {
+      return (
```

Comment:
There is a lot of duplicated code here, you need to only add one class conditionally, not redo the whole message structure.